src/devices/Ads1115/Ads1115.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Ads1115
+{
+    /// <summary>
+    /// Analog-to-Digital Converter ADS1115
+    /// </summary>
+    public class Ads1115 : IDisposable
+    {
+        private I2cDevice _sensor = null;
+
+        private readonly byte _adcMux;
```

Comment:
Can we change these variable names to make them more descriptive?
`_adcMux` -> `_inputMultiplexer`
`_adcPga` -> `_programmableGainAmplifier`
`_adcRate` -> `_dataRate`